apps/studio/components/interfaces/Organization/BillingSettings/Subscription/PaymentMethodSelection.tsx

```diff
@@ -43,19 +42,17 @@ const PaymentMethodSelection = ({
   )
 
   useEffect(() => {
```

Comment:
can you do this in the success callback of useOrganizationPaymentMethodsQuery and save us the indignity of a useEffect?

Reply:
onSuccess is deprecated